File: services/fiuba.py
import requests
import os


class FiubaService:

    URL = os.environ.get("API_FIUBA_URL", "http://www.fi.uba.ar/api/")
    STUDENT_ENDPOINT = "students/"

    def validateStudentId(self, student_id):
        url_endpoint = self.getStudentEnpointURL(student_id)
        response = requests.get(url_endpoint)

        return response.status_code == 200

    def getStudentEnpointURL(self, student_id):
        return self.URL + self.STUDENT_ENDPOINT + student_id

File: src/Services/fiuba.js
"use strict"
const config = require('../Configuration/config');
const fetch = require("node-fetch");


class FiubaService{

    static async validateStudentId(student_id){
        let url_endpoint = this.getStudentEndpointURL(student_id);
        let response = await fetch(url_endpoint);
        return response.ok;
    }

    static getStudentEndpointURL(student_id){
        return config.FIUBA_URL + "students/" + student_id
    }
}

module.exports = FiubaService;
